packages/default/scss/dock-manager/_index.scss

```diff
@@ -0,0 +1,38 @@
+@import "../core/_index.scss";
+@import "../utils/_index.scss";
+
+// Module meta
+$_kendo-module-meta: (
+    name: "dock-manager",
+    dependencies: (
+        "splitter",
+        "toolbar",
+        "tabstrip"
+    )
+);
+
+// Dependencies
```

Comment:
Shouldn't we add @import "../common/_index.scss"; as a dependency?

Reply:
Not needed, as the component does not utilze any module from `common`.